tensorboard/webapp/plugins/npmi/views/annotations_list/annotations_list_toolbar/annotations_list_toolbar_component.scss

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+:host {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  padding: 0 16px;
+  width: calc(100% - 32px);
+}
+
+.annotations-toolbar-not-expanded {
+  align-items: center;
+  background-color: mat-color($mat-light-theme-background, card);
+  border: 1px solid mat-color($tb-foreground, border);
+  display: flex;
+  height: 42px;
+  padding: 0 16px;
```

Comment:
if this `0 16px` is related to L21, you can factor that out as a variable like `$_list-toolbar-padding: 16px` (this SASS pseudo code might be wrong)